### PR TITLE
#2 - added type ahead search capabilities

### PR DESCRIPTION
**Pull request by patel-manas**

### CODE REVIEW

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
-<h1 class="heading">Custom Builders</h1>
+<h1 class="heading">
+  <span>Custom Builders</span>
+  <app-search-bar #sb (onChange) = "onSearch($event)"></app-search-bar>
```
Comment (santoshyadav198613) [inline src/app/dashboard/dashboard.component.html]: can you reomove extra space (onChange)="onSearch($event)"

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+</h1>
 <ng-container *ngIf="builders$ | async as builders">
```
Comment (santoshyadav198613) [inline src/app/dashboard/dashboard.component.html]: you can also remove as it is not getting used.

File: src/app/dashboard/dashboard.component.html (hunk 1 above)
Comment (patel-manas) [reply src/app/dashboard/dashboard.component.html]: @santoshyadav198613  updated as per your comments

**Comment by santoshyadav198613** [comment]
@all-contributors please add patel-manas for code

**Comment by santoshyadav198613** [comment]
@all-contributors please add @patel-manas for code

**Pull request merged**